src/modules/infobox/components/styling/FeatureStyleEdit.vue

```diff
@@ -159,6 +167,55 @@ export default {
                 this.changeFeatureTitle({ feature: this.feature, title: value })
             },
         },
+        geometry: {
+            get() {
+                /*
+                Openlayers polygons coordinates are in a triple array
+                The first array is the "ring", the second is to hold the coordinates, which are in an array themselves
+                We don't have rings in this case, so we need to create an ol geometry
+                */
+
+                const geom = [this.feature.coordinates]
+
+                return new Polygon(geom)
+            },
+        },
+        isClosed: {
+            get() {
+                return (
+                    this.feature.coordinates[0][0] ===
+                        this.feature.coordinates[this.feature.coordinates.length - 1][0] &&
+                    this.feature.coordinates[0][1] ===
+                        this.feature.coordinates[this.feature.coordinates.length - 1][1]
+                )
+            },
+        },
+        length: {
+            get() {
+                const length = getLength(this.geometry)
+
+                let output
+
+                if (length > 100) {
+                    output = Math.round((length / 1000) * 100) / 100 + ' ' + 'km'
+                } else {
+                    output = Math.round(length * 100) / 100 + ' ' + 'm'
+                }
+                return output
```

Comment:
a bit of simplification could go a long way
```
// way above 
import { round } from '@/utils/numberUtils'

// then here
const length = getLength(this.geometry)
if (length > 100) {
    return `${round(length / 1000.0)} km`
}
return `${round(length)} km`
```
Should do about the same as what you have written